src/sbvr-api/hooks.ts

```diff
@@ -343,12 +370,11 @@ export const addPureHook = (
 	});
 };
 
-const defineApi = (args: HookArgs) => {
-	const { request, req, tx } = args;
-	const { vocabulary } = request;
+const defineApi = (modelName: string, args: HookArgs) => {
```

Comment:
Honestly I'm leaning towards just moving everything to using `modelName` instead of `vocabulary` because I think it generally makes more sense, especially because there can technically be multiple vocabularies within a single model..